packages/experimental/Badge/src/Badge.styling.ts

```diff
@@ -30,18 +33,20 @@ export const stylingSettings: UseStylingOptions<BadgeProps, BadgeSlotProps, Badg
     root: buildProps(
       (tokens: BadgeTokens, theme: Theme) => ({
         style: {
+          ...getBadgePosition(tokens),
           display: 'flex',
           alignItems: 'center',
           flexDirection: 'row',
           alignSelf: 'flex-start',
           justifyContent: 'center',
           height: tokens.height,
+          width: tokens.width,
           backgroundColor: tokens.backgroundColor,
           ...borderStyles.from(tokens, theme),
           ...layoutStyles.from(tokens, theme),
         },
       }),
-      ['backgroundColor', 'height', ...borderStyles.keys, ...layoutStyles.keys],
+      ['backgroundColor', 'width', 'height', 'bottom', 'right', ...borderStyles.keys, ...layoutStyles.keys],
```

Comment:
I think top and left should be added here, too

Reply:
added